internal/global/validator/validator.go

```diff
@@ -0,0 +1,21 @@
+package validator
+
+import (
+	"github.com/gin-gonic/gin/binding"
+	"github.com/go-playground/validator/v10"
+	"github.com/lucky-pocket/luckyPocket-back/internal/domain/data/constant"
+)
+
+func Initialize() {
```

Comment:
```suggestion
func Initialize() error {
```
Initialize가 error를 반환하면 좋을 것 같습니다!

Reply:
사실 Initialize가 *validator.Validate 타입을 인수로 받는 것도 괜찮을 것 같습니다. binding 호출은 main같은 바깥에서 하고요

Reply:
https://github.com/lucky-pocket/luckyPocket-back/pull/92/commits/7c631e272273b58eab70b7d99a2c7ec89018c677

Reply:
https://github.com/lucky-pocket/luckyPocket-back/pull/92/commits/82aafb981b33f44c6e2ad6c251202e5eb968c6eb